packages/node_modules/@ciscospark/widget-message/src/actions.js

```diff
@@ -15,15 +14,6 @@ export function updateWidgetState(state) {
   };
 }
 
-export function setScrollPosition(scrollPosition) {
-  return {
-    type: SET_SCROLL_POSITION,
-    payload: {
-      scrollPosition
-    }
-  };
-}
-
```

Comment:
I don't think we can get rid of scroll position. It's used to calculate where the scroll position should be set when new messages come in. 
Looking at the netlify demo, when I scroll to the top of the activity list and more messages load, the view jumps to the oldest message instead of staying in the same spot with the same messages in view.